app/views/indicators/index.html.erb

```diff
@@ -99,14 +99,14 @@
             <div class="c-table-list-item__text f-ff1-m"><%= indicator.category %></div>
           </div>
           <div class="small-2 columns">
-            <div class="c-table-list-item__text f-ff1-m"><%= indicator.stack_family %></div>
+            <div class="c-table-list-item__text f-ff1-m"><%= indicator.subcategory %></div>
           </div>
           <div class="small-1 columns">
             <div class="c-table-list-item__text f-ff1-m"><%= indicator.unit %></div>
           </div>
           <div class="small-4 columns">
             <div class="c-table-list-item__text f-ff1-m">
-              <%= indicator.definition.length > 120 ? truncate(indicator.definition, length: 120) : indicator.definition %>
+              <%= indicator.definition && indicator.definition.length > 120 ? truncate(indicator.definition, length: 120) : indicator.definition %>
```

Comment:
I'm wondering if the length check is necessary here. I believe that the `truncate` method will return the untouched string if its length is <= than the passed length parameter.